### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by avdev4j**
close https://github.com/entando/entando-blueprint/issues/98

### CODE REVIEW

**Comment by avdev4j** [comment]
I wonder if we should remove this methor or not? https://github.com/entando/entando-blueprint/pull/99/files#diff-36160804c5d2c7b6b6ceff1226bc58bbR36

I basically ignore the lint error but if this method not used we should, definitively, remove it.

File: generators/server/phases/writing/index.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ function addMavenSnapshotRepository() {
   this.addMavenRepository('snapshot-repo', 'https://oss.sonatype.org/content/repositories/snapshots');
 }
 
+// TODO Remove this method if we don't need it anymore
+// eslint-disable-next-line
 function addEntandoAuthDependencies() {
```
Comment (Kerruba) [inline generators/server/phases/writing/index.js:36]: I think we should be able to remove this,
All the rest looks good to me :+1: 

Thanks

**Review by Kerruba — CHANGES_REQUESTED** [review]
Simply remove the method that's not used anymore, all the rest is ok to me

File: generators/server/phases/writing/index.js (hunk 1 above)
Comment (avdev4j) [reply generators/server/phases/writing/index.js]: Done ;)

**Review by Kerruba — APPROVED** [review]
LGTM

**Pull request merged**